config/track_test.go

```diff
@@ -14,16 +14,26 @@ func TestTrackIgnoreString(t *testing.T) {
 		},
 	}
 
-	tests := map[string]bool{
+	testCases := map[string]bool{
 		"falcon.txt": false,
 		"beacon|txt": true,
 		"beacon.ext": true,
 		"proof":      false,
 	}
 
-	for name, acceptable := range tests {
-		ok, err := track.AcceptFilename(name)
-		assert.NoError(t, err, name)
-		assert.Equal(t, acceptable, ok, name)
+	for name, acceptable := range testCases {
+		testName := name + " should " + notIfNeeded(acceptable) + "be an acceptable name."
+		t.Run(testName, func(t *testing.T) {
+			ok, err := track.AcceptFilename(name)
+			assert.NoError(t, err, name)
+			assert.Equal(t, acceptable, ok, testName)
+		})
 	}
 }
+
+func notIfNeeded(b bool) string {
```

Comment:
Yes. It's a quick hack because the original code, instead of using a `[]struct input {input, expected_output}` uses a map with a (string) key as the input value, and the value as the expected output.
I think this can just be replaced with the pattern that is used elsewhere, then it would be easy to add a description field to the struct, but I was not.